src/cosmogony.rs

```diff
@@ -35,6 +37,15 @@ impl CosmogonyStats {
             }
         });
     }
+    pub fn process_zone_types(&mut self, zones: &Vec<Zone>) {
+        for zone in zones {
+            let type_ = zone.zone_type
+                .unwrap_or(ZoneType::NonAdministrative)
```

Comment:
I'm not very proud of this. Any idea to deal with it in a better way ? :thinking:

Reply:
and if you just change `zone_type_counts` to be `BTreeMap<Option<ZoneType>, u64>` ?

this way we would have stats on unknown zone_types no ?